src/main/java/net/authorize/api/controller/base/ApiOperationBase.java

```diff
@@ -252,7 +252,7 @@ private void validateAndSetMerchantAuthentication() {
 		}
 	}
 	
-	public synchronized String getClientId() {
+	public String getClientId() {
 		return Constants.CLIENT_ID;

```

Comment:
This was added in 1.9.0. Verified that this was not causing any issue. Removed synchronized just to be safe.